tests/dragonfly/json_test.py

```diff
@@ -14,90 +15,80 @@
 }
 
 
-def get_type(connection, key):
-    return connection.execute_command("type", key)
+async def get_set_json(connection: aioredis.Redis, key, value, path="$"):
+    encoder = JSONEncoder()
+    await connection.execute_command("json.set", key, path, encoder.encode(value))
+    result = await connection.execute_command("json.get", key, path)
+    decoder = JSONDecoder()
+    return decoder.decode(result)
```

Comment:
You can just use json.dumps and json.loads

Reply:
next time :)